addons/web/static/src/js/control_panel/custom_filter_item.js

```diff
@@ -118,9 +118,9 @@ odoo.define('web.CustomFilterItem', function (require) {
                     }
                     break;
                 case 'datetime':
-                    condition.value = [moment('00:00:00', 'hh:mm:ss')];
+                    condition.value = [moment('00:00:00', 'hh:mm:ss').utcOffset(0, true)];
```

Comment:
Why not directly set the value in UTC? (with `moment.utc('00:00:00', 'hh:mm:ss')`)

Reply:
Because in some cases, it might pick the wrong day, according to @nle-odoo ;)